README.md

```diff
@@ -125,6 +123,4 @@ Code reviewers were of great help
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] The link to your LICENSE is broken and redirects to GitHub's error page. Kindly fix the LICENSE. You can add a valid LICENSE by following the instructions in this [article](https://choosealicense.com/licenses/mit/). 👍 😎